src/coreclr/src/vm/pefile.cpp

```diff
@@ -2475,3 +2474,116 @@ PTR_ICLRPrivBinder PEFile::GetBindingContext()
 
     return pBindingContext;
 }
+
+// Function to get the fully qualified name of an assembly
+void PEAssembly::GetFullyQualifiedAssemblyName(IMDInternalImport* pImport, mdAssembly mda, SString& result, DWORD flags)
```

Comment:
Copy & paste from the .inl file to avoid including `strongnameinternal.h` globally.